clang/lib/CodeGen/CGRecordLayoutBuilder.cpp

```diff
@@ -743,6 +743,7 @@ void CGRecordLowering::calculateZeroInit() {
 void CGRecordLowering::clipTailPadding() {
   std::vector<MemberInfo>::iterator Prior = Members.begin();
   CharUnits Tail = getSize(Prior->Data);
+
```

Comment:
Nit: The total PR sums up to adding a spurious empty line here, while this file otherwise no longer has any functional changes.